src/game/client/languagefilter.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief Language Filter
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "languagefilter.h"
+
+#ifndef GAME_DLL
+LanguageFilter *g_theLanguageFilter;

```

Comment:
Does this need to be initialized to something?

Reply:
no